ecommerce/admin.py

```diff
@@ -314,6 +322,12 @@ class ProductAdmin(admin.ModelAdmin):
     inlines = [ProductVersionInline]
     list_display = ("id", "content_object")
     list_filter = ("is_active", ProductContentTypeListFilter)
+    search_fields = (
+        "courseruns__title",
+        "programs__title",
+        "courseruns__courseware_id",
+        "programs__readable_id",
```

Comment:
Same here can we replace `product__courseruns__courseware_id` and `product__programs__readable_id` with `text_id`.

Reply:
True but it isn't hard-wired to be that. It's a nullable text field so I wanted it to be explicit.